example/starter.script

```diff
@@ -0,0 +1,54 @@
+function init(self)
+	msg.post(".", "acquire_input_focus")
+  msg.post("#gui_proxy", "load")
+  print("msg.url ",msg.url("#"))
+end
+
+function final(self)
+	-- Add finalization code here
+	-- Learn more: https://defold.com/manuals/script/
+	-- Remove this function if not needed
+end
+
+function update(self, dt)
+	-- Add update code here
+	-- Learn more: https://defold.com/manuals/script/
+	-- Remove this function if not needed
+end
+
+
+function on_message(self, message_id, message, sender)
+  local launcher = {
```

Comment:
This declaration should be moved outside in the on_message function. If you keep it inside the function it will create a new table each time the function is called.